metrics/config_observability.go

```diff
@@ -83,6 +87,16 @@ func NewObservabilityConfigFromConfigMap(configMap *corev1.ConfigMap) (*Observab
 		return nil, err
 	}
 
+	if raw, ok := configMap.Data["logging.enable-request-log"]; ok {
+		if strings.EqualFold(raw, "true") && oc.RequestLogTemplate != "" {
```

Comment:
I could start with `true` in the default config above for the flag and then parse it along with the others but this way it is more clear.